### PR TITLE
Feature/mp 137/create glue resources

### PR DESCRIPTION
**Pull request by a-sabaa**

### CODE REVIEW

File: config/spryk/templates/Glue/BackOffice/GlueBackOfficeRestResourceRelationshipPlugin.php.twig [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    /**
+     * @param array $keys
+     * @param \Spryker\Glue\GlueRestApi\Rest\Request\Data\RestRequest $restRequest
+     *
+     * @return \Spryker\Glue\GlueRestApi\Rest\JsonApi\RestResourceCollectionInterface
+     */
+    public function fetchRelationshipData(array $keys, RestRequest $restRequest): RestResourceCollectionInterface
+    {
+
```
Comment (stereomon) [inline config/spryk/templates/Glue/BackOffice/GlueBackOfficeRestResourceRelationshipPlugin.php.twig]: Remove empty line, please.

File: config/spryk/templates/Glue/GlueController.php.twig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+namespace {{ organization }}\Glue\{{ module }}\Controller;
+
+use Spryker\Glue\Kernel\Controller\AbstractController;
+
+/**
+ * @method \{{ organization }}\Glue\{{ module }}\{{ module }}ResourceInterface getResource()
+ */
+class {{ className }} extends AbstractController
+{
+
```
Comment (stereomon) [inline config/spryk/templates/Glue/GlueController.php.twig]: Remove empty line, please.

File: config/spryk/templates/Glue/Storefront/GlueStorefrontRestResourceRelationshipPlugin.php.twig [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    /**
+     * @param array $keys
+     * @param \Spryker\Glue\GlueRestApi\Rest\Request\Data\RestRequest $restRequest
+     *
+     * @return \Spryker\Glue\GlueRestApi\Rest\JsonApi\RestResourceCollectionInterface
+     */
+    public function fetchRelationshipData(array $keys, RestRequest $restRequest): RestResourceCollectionInterface
+    {
+
```
Comment (stereomon) [inline config/spryk/templates/Glue/Storefront/GlueStorefrontRestResourceRelationshipPlugin.php.twig]: Remove empty line, please.

File: tests/_support/Module/CleanUpModule.php [hunk 4]
```diff
@@ -25,6 +25,24 @@ class CleanUpModule extends Module
...
+    {
+        $this->removeCreatedFiles();
+    }
+
+    /**
+     * @param \Codeception\TestInterface $test
+     *
+     * @return void
+     */
+    public function _after(TestInterface $test): void
```
Comment (stereomon) [inline tests/_support/Module/CleanUpModule.php]: When the test fails you can not look into the generated file anymore as it is directly removed. That's the reason why you should always clean up before the tests.

**Comment by matweew** [comment]
@a-sabaa is it still relevant? I see Glue Resource already exists in the Spryk

**Comment by stereomon** [comment]
Is this one still required or can we close it?

**Pull request closed without merging**